runelite-mixins/src/main/java/net/runelite/mixins/RSClientMixin.java

```diff
@@ -1338,10 +1344,30 @@ public static void renderWidgetLayer(Widget[] widgets, int parentId, int var2, i
 			{
 				widget.setRenderParentId(parentId);
 			}
-			widget.setRenderX(x + widget.getRelativeX());
-			widget.setRenderY(y + widget.getRelativeY());
 
-			HashTable<WidgetNode> componentTable = client.getComponentTable();
+			final int renderX = x + widget.getRelativeX();
+			final int renderY = y + widget.getRelativeY();
+			widget.setRenderX(renderX);
+			widget.setRenderY(renderY);
+
+			final int widgetType = widget.getType();
+			if (widgetType == WidgetType.GRAPHIC && widget.getItemId() != -1)
```

Comment:
You probably want to also add the check for if this is the dragged widget `parentID == -1412584499`